src/creature/main_character/main_character.cpp

```diff
@@ -38,8 +38,9 @@ MainCharacter::MainCharacter(std::string name,
 	int intelligence,
 	int wisdom,
 	int charisma,
-	std::string creature_type)
-	: Creature(hp, ac, speed, strength, dexterity, constitution, intelligence, wisdom, charisma, creature_type),
+	std::string creature_type,
+	std::string display_name)
```

Comment:
I don't believe we will ever need to create a `MainCharacter` where we supply the display name. It will always be "main character".